Style conformance broken

The style conformance is broken which means tests will not run in a dev environment.

```console
$ make test
/home/usr1/go/bin/golangci-lint run
pkg/kube/fake/fake.go:35: File is not `goimports`-ed (goimports)
	CreateError error
	WaitError error
	GetError error
	DeleteError error
	WatchUntilReadyError error
	UpdateError error
	BuildError error
	BuildUnstructuredError error
cmd/helm/history.go:23: File is not `goimports`-ed (goimports)
	"github.com/spf13/cobra"
pkg/action/uninstall.go:128: File is not `goimports`-ed (goimports)
		
Makefile:81: recipe for target 'test-style' failed
make: *** [test-style] Error 1
```

--- a/cmd/helm/history.go
+++ b/cmd/helm/history.go
@@ -20,14 +20,14 @@ import (
 	"fmt"
 	"io"
 
-	"github.com/spf13/cobra"
 	"github.com/gosuri/uitable"
+	"github.com/spf13/cobra"
 
 	"helm.sh/helm/cmd/helm/require"
 	"helm.sh/helm/pkg/action"
-	"helm.sh/helm/pkg/releaseutil"
 	"helm.sh/helm/pkg/chart"
 	"helm.sh/helm/pkg/release"
+	"helm.sh/helm/pkg/releaseutil"
 )
 
 var historyHelp = `
@@ -131,7 +131,6 @@ func getHistory(client *action.History, name string) (string, error) {
 
 	return string(history), nil
 }
-
 
 func getReleaseHistory(rls []*release.Release) (history releaseHistory) {
 	for i := len(rls) - 1; i >= 0; i-- {

--- a/pkg/action/uninstall.go
+++ b/pkg/action/uninstall.go
@@ -125,12 +125,12 @@ func (u *Uninstall) Run(name string) (*release.UninstallReleaseResponse, error) 
 		if err != nil {
 			errs = append(errs, errors.Wrap(err, "uninstall: Failed to purge the release"))
 		}
-		
+
 		// Return the errors that occurred while deleting the release, if any
 		if len(errs) > 0 {
 			return res, errors.Errorf("uninstallation completed with %d error(s): %s", len(errs), joinErrors(errs))
 		}
-		
+
 		return res, nil
 	}
 

--- a/pkg/kube/fake/fake.go
+++ b/pkg/kube/fake/fake.go
@@ -32,14 +32,14 @@ import (
 // delegates all its calls to `PrintingKubeClient`
 type FailingKubeClient struct {
 	PrintingKubeClient
-	CreateError error
-	WaitError error
-	GetError error
-	DeleteError error
-	WatchUntilReadyError error
-	UpdateError error
-	BuildError error
-	BuildUnstructuredError error
+	CreateError                      error
+	WaitError                        error
+	GetError                         error
+	DeleteError                      error
+	WatchUntilReadyError             error
+	UpdateError                      error
+	BuildError                       error
+	BuildUnstructuredError           error
 	WaitAndGetCompletedPodPhaseError error
 }
 

